packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -53,6 +54,34 @@ const resolveSideEffects = () => ({
 	notFound: NotFound.rewind(),
 });
 
+/**
+ * Using the User agent string + an external library called Mobile-Device we detect if the user
+ * is attempting to render the page on the server using a mobile, tablet, or desktop device
+ * @param  {String} userAgent User agent string detected from headers
+ * @return {Object} Object of which category the devices falls within
+ */
+const getMedia = (userAgent: string ) => {
+	const isAtSmallUp = true;
+	// we may be re-writing the User-Agent header in Fastly for caching purposes, so check that first
+	if (userAgent === 'mobile' || userAgent === 'desktop') {
+		return {
+			isAtSmallUp,
+			isAtMediumUp: userAgent !== 'mobile',
+			isAtLargeUp: userAgent === 'desktop'
+		};
+	}
+	// parses user agent string to determine if user is on mobile / tablet / desktop device
+	const device = new MobileDetect(userAgent);
+	const isMobile = device.mobile() !== null && device.phone() !== null;
```

Comment:
is this sufficient?
```js
const isMobile = Boolean(device.phone())
```

if `device.phone()` returns a value, then do you _also_ need to know that `device.mobile()` returns a value?